PendingReleaseNotes.md

```diff
@@ -31,8 +45,8 @@
   - Standalone mode has been disabled and is no longer supported. A Kubernetes environment is required to run Rook.
 - Rook-operator now deploys in `rook-system` namespace
   - If using the example manifest of [rook-operator.yaml](/cluster/examples/kubernetes/rook-operator.yaml), the rook-operator deployment is now changed to deploy in the `rook-system` namespace.
-- Introduces a new flexvolume plugin that will handle all volume attachment requests. This will simplify the deployment by not requiring to have ceph-tools package installed.
-- For Kubernetes cluster 1.7.x and older, Kubelet will need to be restarted in order to load the new flexvolume. This has been resolve in K8S 1.8. For more information about the requirements, refer to this [doc](/Documentation/k8s-pre-reqs.md)
+- Rook Flexvolume
+  - Previously created PVCs were created using the RBD plugin. These should be deleted and recreated in order to use the next Flexvolume plugin.
```

Comment:
could be phrased better as: "Persistent volumes from previous releases were created using the RBD plugin..."

"new" Flexvolume plugin instead of "next"?